template/package.json

```diff
@@ -6,7 +6,7 @@
     "start": "node src/api/index.js"
   },
   "dependencies": {
```

Comment:
please remove `^` from all dependencies. Since we do not have package-lock here and this whole storm with color.js popped up, IMHO safe to remove `^` and do not get surprised by patches

Reply:
Done